grails-app/taglib/org/pih/warehouse/AuthTagLib.groovy

```diff
@@ -43,6 +43,18 @@ class AuthTagLib {
         }
     }
 
+    def isNonInventoryManaged = {attrs, body ->
```

Comment:
Technically the sole `isNonInventoryManaged` is not true here because it needs `SUBMIT_REQUEST` too. `isNonInventoryManagedAndCanSubmitRequest` would be better in this case. However, I am thinking now that it might be useful for other places to add a tag `notSupports` and then in the template do:
```
<!-- Check if location does NOT support MANAGE_INVENTORY, but supports SUBMIT_REQUEST, then display no menu -->
<g:notSupports activityCode="${ActivityCode.MANAGE_INVENTORY}">
    <g:supports activityCode="${ActivityCode.SUBMIT_REQUEST}">
        <g:if test="${megamenuConfig.dashboard.enabled}">
            <g:isUserInRole roles="[RoleType.ROLE_BROWSER]">
                <li class="mm-item">
                    <g:link controller="dashboard" action="index" class="mm-item-link">
                        <warehouse:message code="dashboard.label" />&nbsp;
                    </g:link>
                </li>
            </g:isUserInRole>
        </g:if>
    </g:supports>
</g:notSupports>
```

Reply:
yes, at first I thought about the same solution and it would have been easier if condition for **rendering** the menu was 

if does **NOT support** _MANAGE_INVENTORY_  [AND]()  supports _SUBMIT_REQUEST_ => **render menu**
else => don't render

but I have to consider the opposite, if this condition is met then don't render the menu which would mean that I have to negate the condition which results in

not( not support(_MANAGE_INVENTORY_) and  support(_SUBMIT_REQUEST_)  )
 => 
support(_MANAGE_INVENTORY_) or  not support(_SUBMIT_REQUEST_)  

and I am not entirely sure how to implement this **OR** logic for **supports or does not support**

that's why I chose to create a specific checker function for this case